src/macros-by-example.md

```diff
@@ -326,6 +326,77 @@ fn foo() {
 // m!(); // Error: m is not in scope.
 ```
 
+r[macro.decl.scope.textual.shadow.path-based]
+Textual scope name bindings for macros may shadow path-based scope bindings to macros.
```

Comment:
The use of the word "may" here feels like it is weakening the statement, or making it sound like it is optional. I'm not sure if that was intentional or if that word is needed here? That is, could it just be dropped?


```suggestion
Textual scope name bindings for macros shadow path-based scope bindings to macros.
```